src/openrct2/windows/editor_bottom_toolbar.c

```diff
@@ -271,7 +253,17 @@ void window_editor_bottom_toolbar_jump_forward_from_object_selection()
 		return;
 
 	if (gScreenFlags & SCREEN_FLAGS_TRACK_DESIGNER) {
-		sub_66F6E3();
+		for (sint32 i = 0; i < 4; i++) {
```

Comment:
I think this line (up to and including 262) should be in a function (if one doesn't exist already). That function should not be in this window, but in the regular research file. (This can be put into a separate issue, of course.)